cfme/utils/appliance/__init__.py

```diff
@@ -3187,6 +3187,11 @@ def create_view(self, view_class, o=None, override=None):
         return self.appliance.browser.create_view(
             view_class, additional_context={'object': new_obj})
 
+    def create_and_wait_view(self, view_class, o=None, override=None, timeout='10s'):
```

Comment:
Thought:

Did you consider implementation of `create_view(wait=None)`, where `wait` kwarg can be set to a timeout value. Existing `create_view` just modified to check for `wait` to be not-`None` and wait_displayed the given number of seconds.

Reply:
this was just a light bulb moment. I hadn't considered embedding it in existing call because it would have required some analysis.

Do you think it's worth replacing it with parameter ?